### PR TITLE
[8.0] the TransformationCleaningAgent forces jobs to be killed

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*TS
CHANGE: the TransformationCleaningAgent forces jobs to KILLED

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py [hunk 1]
```diff
@@ -721,6 +719,11 @@ def __submitRemovalRequests(self, lfns, transID=0):
         :param int transID: transformationID, only used in RequestName
         :returns: S_ERROR/S_OK
         """
+
+        # These imports are used only in this function
```
Comment (chrisburr) [inline src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py:723]: Why?

Comment (fstagni) [reply src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py:723]: These imports are used a few lines below in this 

```
requestName = "TCA_{transID}_{index}_{md5(repr(time.time()).encode()).hexdigest()[:5]}"
```

given that I do not want to see in the future the ruff warning that these are useless imports, I moved them here.

Comment (chrisburr) [reply src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py:723]: I don't see what the problem is with them being at the top of the file? Why are these imports so special that they're worth moving to the local scope? `errno` is also only used in a single place.

Comment (andresailer) [reply src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py:723]: I guess ruff doesn't parse the use inside the f-string?

Comment (fstagni) [reply src/DIRAC/TransformationSystem/Agent/TransformationCleaningAgent.py:723]: > I guess ruff doesn't parse the use inside the f-string?

Exactly. And the next time I see them, if I do not pay attention I remove them.

**Review by atsareg — APPROVED** [review]

**Pull request merged**